crawl4ai/async_crawler_strategy.py

```
@@ -447,17 +464,28 @@ async def crawl(
                 html = f.read()
             if config.screenshot:
                 screenshot_data = await self._generate_screenshot_from_html(html)
+            if config.capture_console_messages:
+                page, context = await self.browser_manager.get_page(crawlerRunConfig=config)
```

Ruff (0.14.0)

Unpacked variable `context` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)